src/components/manifold-subscription-details/manifold-subscription-details.tsx

```diff
@@ -41,26 +42,9 @@ export class ManifoldSubscriptionCreate {
       if (response.data) {
         this.data = response.data;
 
-        const normalizeValue = (node: any) => {
-          if (node.stringValue !== undefined) {
-            return node.stringValue;
-          }
-          if (node.numberValue !== undefined) {
-            return node.numberValue;
-          }
-          if (node.booleanValue !== undefined) {
-            return node.booleanValue;
-          }
-          return undefined;
-        };
-        const featureMap = response.data.subscription.configuredFeatures.edges.reduce(
-          (features, { node }) => ({
-            ...features,
-            [node.label]: normalizeValue(node),
-          }),
-          {}
+        state.configuredFeatures = toFeatureMap(
+          response.data.subscription.configuredFeatures as any
```

Comment:
These types are kind of annoying when you want to be able to use this with data from different queries :/  maybe it's best to just build the type up somewhat manually? or maybe this would be a good use for fragments. Idk how to reuse fragments across .graphql files though

Reply:
Is it not a `ConfiguredFeatureConnection`?

Reply:
it won't even let me cast it as `ConfiguredFeatureConnection `